UI/Tooltips/AchievementTooltipView.cs

```diff
@@ -110,5 +111,27 @@ public class AchievementTooltipView : View, ITooltipView {
             _achievementRequirementLabel.Top = Math.Max(_achievementDescriptionLabel.Bottom + 8, _categoryIconImage.Bottom + 8);
         }
 
+        static string CleanMessage(string message) {
+            // Perform manipulations on the cleaned message
+            string cleanedMessage = message;
+
+            // This Regex catches both the color and raw text of colored segments
+            string pattern = @"<c[=@][@=]?([^>]+)>(.*?)(<\/?c\/?>|$)";
```

Comment:
For now these will work defined here, but if we later change to pulling this data all at once and applying this to the whole data set or something, we'll want to precompile these expressions.

Reply:
I could be wrong here, but I believe that using the static `Regex.Matches` (or any of those static functions) actually caches the compiled regex expression. 

https://docs.microsoft.com/en-us/dotnet/standard/base-types/compilation-and-reuse-in-regular-expressions

> ## The Regular Expressions Cache
> To improve performance, the regular expression engine maintains an application-wide cache of compiled regular expressions. The cache stores regular expression patterns that are used only in static method calls. (Regular expression patterns supplied to instance methods are not cached.) This avoids the need to reparse an expression into high-level byte code each time it is used.

Using a completely static instance may be slightly better (avoids the lookup) but I doubt it'd be a noticeable difference.